src/std/error.ss

```diff
@@ -2,6 +2,7 @@
 ;;; (C) vyzo
 ;;; Gerbil error objects
 (import :gerbil/runtime/error
+        :gerbil/runtime/thread
```

Comment:
Do we need this?

Reply:
yes, we reexport with-exception-stack-trace.